Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -272,6 +276,23 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 		StatStopFileTimer.SetTimer(999999);
 	}
 #endif
+	if (MemReportIntervalTimer.HasTimerGoneOff())
+	{
+		if (MemReportInterval>0) //Avoid execute memreport each tick before the worker flag take effect
```

Comment:
Why don't we check `MemReportInterval > 0` first, since it looks like a switch of this feature?